src/publish.proj

```diff
@@ -18,13 +18,14 @@
 
   <PropertyGroup>
     <ContainerName>$(ContainerName.Replace(".","-"))</ContainerName>
+    <PublishBlobFeedUrl>$(PB_PublishBlobFeedUrl)<PublishBlobFeedUrl>
     <AccountName>$(CloudDropAccountName)</AccountName>
     <AccountKey>$(CloudDropAccessToken)</AccountKey>
   </PropertyGroup>
 
   <Target Name="PublishPackages" Condition="'$(__PublishPackages)' == 'true' and ('$(OfficialPublish)' != 'true' or '$(__BuildType)' == 'Release')">
     <PropertyGroup>
-      <ExpectedFeedUrl>https://$(AccountName).blob.core.windows.net/$(ContainerName)/$(RelativePathWithSlash)index.json</ExpectedFeedUrl>
+      <ExpectedFeedUrl>$(PublishBlobFeedUrl)</ExpectedFeedUrl>
```

Comment:
This is going to break the non-publish build legs. I've been wanting those legs to pass this in as well so you could go ahead and do that.

Reply:
You still need to fix the publish steps in the individual legs